CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 
 # Changelog
 
+## [22.09.1]
+* [UI]: Fixed bug preventing sharing samples via the linelist page. See [PR 1382](https://github.com/phac-nml/irida/pull/1382)
```

Comment:
Can you update this comment to include a bit on the the various other minor bugs. `For cart page can just say fixed visual bug, or something like that).